example.ini

```diff
@@ -30,6 +34,11 @@ factory = reddit_service_activitygateway:make_wsgi_app
 metrics.namespace = activity_gateway
 metrics.endpoint =
 
+; request tracing
+tracing.enabled = true
```

Comment:
I wanted to provide flexibility for turning on trace calculation and management without needing to specify an endpoint for it (e.g. still logging locally). See https://github.com/reddit/baseplate/blob/master/baseplate/diagnostics/tracing.py#L60 for where I already branch on whether an endpoint exists for remote recording.